modules/host/contractcreation.go

```diff
@@ -119,14 +113,7 @@ func verifyContract(contract consensus.FileContract, terms modules.ContractTerms
 func (h *Host) acceptContract(txn consensus.Transaction) error {
 	contract := txn.FileContracts[0]
 	duration := uint64(contract.Expiration - contract.Start)
-
-	penalty := h.Collateral
-	penalty.Mul(consensus.NewCurrency64(contract.FileSize))
-	err := penalty.Mul(consensus.NewCurrency64(duration))
-	// TODO: move this check to a different function?
-	if err != nil {
-		return err
-	}
+	penalty := h.Collateral.Mul(consensus.NewCurrency64(contract.FileSize)).Mul(consensus.NewCurrency64(duration))
```

Comment:
hmm. I don't see a great way to shorten this. One way would be to do the Currency conversions beforehand:
```go
filesizeCost, durationCost := consensus.NewCurrency64(contract.FileSize), (consensus.NewCurrency64(duration)
penalty := h.Collateral.Mul(filesizeCost).Mul(durationCost)
```
The other way would be to split the `Mul`s apart:
```go
penalty := h.Collateral
penalty = penalty.Mul(consensus.NewCurrency64(contract.FileSize))
penalty = penalty.Mul(consensus.NewCurrency64(duration))
```
I definitely prefer the former to the latter, but `filesizeCost` and `durationCost` are awkward names.